diff --git a/r3dg-rasterization/cuda_rasterizer/rasterizer_impl.h b/r3dg-rasterization/cuda_rasterizer/rasterizer_impl.h
index 35ec9d1..8cccc05 100644
--- a/r3dg-rasterization/cuda_rasterizer/rasterizer_impl.h
+++ b/r3dg-rasterization/cuda_rasterizer/rasterizer_impl.h
@@ -1,3 +1,4 @@
+#include <cstdint>
 /*
  * Copyright (C) 2023, Inria
  * GRAPHDECO research group, https://team.inria.fr/graphdeco
diff --git a/submodules/simple-knn/simple_knn.cu b/submodules/simple-knn/simple_knn.cu
index e72e4c9..df436d0 100644
--- a/submodules/simple-knn/simple_knn.cu
+++ b/submodules/simple-knn/simple_knn.cu
@@ -1,3 +1,4 @@
+#include <cfloat>
 /*
  * Copyright (C) 2023, Inria
  * GRAPHDECO research group, https://team.inria.fr/graphdeco
